apps/hash-graph/lib/graph/src/api/error.rs

```diff
@@ -0,0 +1,25 @@
+use std::collections::HashMap;
+
+use hash_status::Status as HashStatus;
+use serde::{Deserialize, Serialize};
+
+pub(super) use crate::api::gen::status_payloads::{ErrorInfo, RequestInfo, ResourceInfo};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum StatusPayloads {
+    ErrorInfo(ErrorInfo),
+    RequestInfo(RequestInfo),
+    ResourceInfo(ResourceInfo),
+}
```

Comment:
Shouldn't these be generated?

Reply:
I think the enum should be generated but it doesn't preserve the actual previous definitions in my experience. It will instead create new duplicative definitions for each variant like we've experienced with Utoipa before.

There are potentially ways around this but it might involve upgrading the `quicktype` version (which we can't do right now because it requires an upgraded `node`) and it might involve swapping to JSON schema definitions instead of TS ones